### PR TITLE
[EDU-3943] feat: add console kit reference

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Added console kit page and installation guide

### Additional links

Menu: #740

[EDU-3000]: https://aziontech.atlassian.net/browse/EDU-3000?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[EDU-3943]: https://aziontech.atlassian.net/browse/EDU-3943?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/devtools/console-kit/console-kit.mdx [hunk 1]
```diff
@@ -0,0 +1,113 @@
+---
+title: Console Kit
+description: Azion Console Kit is a front-end development kit used for crafting a customized Azion Console interface. Developed with Vue/Vite, PrimeVue, and Tailwind, this kit is integrated with the Azion edge computing platform.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/console-kit/console-kit.mdx:3]:
```suggestion
description: Developed with Vue/Vite, PrimeVue, and Tailwind, Azion Console Kit is a front-end development kit used for crafting a customized Azion Console interface.
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/devtools/console-kit/console-kit.mdx:3]: Shorter description.

File: src/content/docs/en/pages/devtools/console-kit/console-kit.mdx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+meta_tags: edge, console, development, interface, vue, primevue
+namespace: documentation_devtools_console_kit
+menu_namespace: devtoolsMenu
+permalink: /documentation/devtools/console-kit/
+---
+
+**Console Kit** is a front-end development kit used for crafting a customized Azion Console interface. Developed with the [Vue/Vite](https://vuejs.org/) and [Tailwind](https://tailwindcss.com/) frameworks, as well as the [PrimeVue](https://primevue.org/) component library, this kit is built upon the [Azion API](https://api.azion.com/) and supports multi-tenancy, which allows you to integrate the Azion edge computing platform with your own APIs.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/console-kit/console-kit.mdx:10]:
```suggestion
Azion **Console Kit** is a front-end development kit used for crafting a customized Azion Console interface. It's developed with the [Vue/Vite](https://vuejs.org/) and [Tailwind](https://tailwindcss.com/) frameworks, as well as the [PrimeVue](https://primevue.org/) component library. This kit is built upon the [Azion API](https://api.azion.com/) and supports multi-tenancy, which allows you to integrate the Azion Edge Computing Platform with your own APIs. 
```

File: src/content/docs/en/pages/devtools/console-kit/console-kit.mdx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+The Console Kit is available on the [Github repository](https://github.com/aziontech/azion-console-kit).
+
+---
+
+## Implementation
+
+Check the full list of requirements and installation and configuration guides in the [Console Kit repository](https://github.com/aziontech/azion-console-kit?tab=readme-ov-file#requirements) on GitHub.
+
+### Install requirements
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/console-kit/console-kit.mdx:22]:
```suggestion
### Requirements
```

File: src/content/docs/en/pages/devtools/console-kit/console-kit.mdx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+:::note
+You may need to rollback Node.js to a previous version. To do so, run:
+
+```bash
+sudo npm install -g n && n 18.16.1
+```
+:::
+
+4. Now run the following command to install the [Yarn](https://yarnpkg.com) package manager:
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/console-kit/console-kit.mdx:39]:
```suggestion
3. Now run the following command to install the [Yarn](https://yarnpkg.com) package manager:
```

File: src/content/docs/en/pages/devtools/console-kit/console-kit.mdx [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+This command should be run periodically to ensure all dependencies are up to date.
+:::
+
+4. Run this command to launch the project locally:
+
+```bash
+yarn dev --host
+```
+
+### Configure environment variables
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/console-kit/console-kit.mdx:75]:
```suggestion
### Configuring environment variables
```

File: src/content/docs/en/pages/devtools/console-kit/console-kit.mdx [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+echo 'VITE_PERSONAL_TOKEN=PERSONALTOKEN' > .env.development
+```
+
+By default, Azion Console Kit uses the `STAGE` stack to connect with Azion APIs. To point your application to the `PRODUCTION` stack, add the following command in the `.env.development` file:
+
+```txt title=".env.development"
+VITE_ENVIRONMENT='PRODUCTION'
+```
+
+### Deploy the Console Kit to the edge
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/console-kit/console-kit.mdx:89]:
```suggestion
### Deploying the Console Kit to the edge
```

File: src/content/docs/en/pages/devtools/console-kit/console-kit.mdx [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+
+Azion Console Kit can run natively on Azion's edge using the [Azion CLI](/en/documentation/products/cli/) (version >= 0.70.0). Follow these steps:
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/console-kit/console-kit.mdx:91]:
```suggestion
Azion **Console Kit** can run natively on Azion's edge using the [Azion CLI](/en/documentation/products/cli/) (version >= 0.70.0). To do so, follow these steps:
```

File: src/content/docs/pt-br/pages/devtools/console-kit/console-kit.mdx [hunk 8]
```diff
@@ -0,0 +1,113 @@
+---
+title: Console Kit
+description: O Azion Console Kit é um kit de desenvolvimento front-end usado para criar uma interface personalizada do Console Azion. Desenvolvido com Vue/Vite, PrimeVue e Tailwind, este kit é integrado à plataforma de computação de borda da Azion.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/console-kit/console-kit.mdx:3]:
```suggestion
description: Desenvolvido com Vue/Vite, PrimeVue e Tailwind, o Azion Console Kit é um kit de desenvolvimento front-end usado para criar uma interface personalizada do Console Azion.
```

File: src/content/docs/pt-br/pages/devtools/console-kit/console-kit.mdx [hunk 9]
```diff
@@ -0,0 +1,115 @@
+---
+title: Console Kit
+description: Desenvolvido com Vue/Vite, PrimeVue e Tailwind, o Azion Console Kit é um kit de desenvolvimento front-end usado para criar uma interface personalizada do Console Azion.
+meta_tags: edge, console, desenvolvimento, interface, vue, primevue
+namespace: documentation_devtools_console_kit
+menu_namespace: devtoolsMenu
+permalink: /documentacao/devtools/console-kit/
+---
+
+O **Console Kit** da Azion é um kit de desenvolvimento front-end usado para criar uma interface personalizada do Console da Azion. Desenvolvido com os frameworks [Vue/Vite](https://vuejs.org/) e [Tailwind](https://tailwindcss.com/), além da biblioteca de componentes [PrimeVue](https://primevue.org/). Este kit é construído sobre a [API da Azion](https://api.azion.com/) e suporta multi-tenancy, permitindo integração da Plataforma de Edge Computing da Azion com as suas próprias APIs.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/console-kit/console-kit.mdx:10]:
```suggestion
O **Console Kit** da Azion é um kit de desenvolvimento front-end usado para criar uma interface personalizada do Console da Azion. Foi desenvolvido com os frameworks [Vue/Vite](https://vuejs.org/) e [Tailwind](https://tailwindcss.com/), além da biblioteca de componentes [PrimeVue](https://primevue.org/). Este kit é construído sobre a [API da Azion](https://api.azion.com/) e suporta multi-tenancy, permitindo integração da Plataforma de Edge Computing da Azion com as suas próprias APIs.
```

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**